Include/Python.h

```diff
@@ -150,4 +150,6 @@
 #include "cpython/pyfpe.h"
 #include "tracemalloc.h"
 
+PyAPI_DATA(long int) Py_Version;
+
```

Comment:
Please guard things that are new in the 3.11+ limited API with:
```c
#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030B0000
```
See https://devguide.python.org/c-api/#adding-a-new-definition-to-the-limited-api

Reply:
Also, consider making it `const`?

Reply:
@encukou thanks for the comments! I have made the requested changes.